mapper/tedge_mapper/src/sm_c8y_mapper/topic.rs

```diff
@@ -0,0 +1,102 @@
+use json_sm::{SoftwareListRequest, SoftwareUpdateRequest};
+use mqtt_client::Topic;
+
+#[derive(Debug, Clone, PartialEq)]
+pub(crate) enum SoftwareCommand {
```

Comment:
The name is slightly misleading and it's hard to identify what this enum represents. I believe what you wanted to capture in this enum are all the topics from which the mapper would receive messages, right? In that case, naming is something like `SoftwareManagementIncomingTopic` would have made it clearer I feel. Just a suggestion.